lib/migrations/20200321153000-fix-account-deletion.js

```diff
@@ -3,7 +3,7 @@ const { spawnSync } = require('child_process')
 const path = require('path')
 module.exports = {
   up: function (queryInterface, Sequelize) {
-    const cleanup = spawnSync('./bin/cleanup', { cwd: path.resolve(__dirname, '../../') })
+    const cleanup = spawnSync(process.argv[0], ['./bin/cleanup'], { cwd: path.resolve(__dirname, '../../') })
```

Comment:
Is this secure to rely on `process.argv[0]`? Or better said: Why not hardcode `node` directly?
If I remember correctly there was once a security problem where argv[0] could be overridden.

Reply:
Fair point, it could be overwritten. The problem with putting `"node"` there, is that it assumes that PATH evaluation is done and that the binary is actually called `node` and not `nodejs` or `node16`. On the other hand, this wasn't handled by the currently used script either, so I guess it's fair to try 🤔

On the other hand, this is called before any user-workload is processed, therefore if this was manipulated, it would have happened in our own code or one of our dependencies, which already has the ability to run code anyway 👀

Reply:
>  if this was manipulated, it would have happened in our own code or one of our dependencies, which already has the ability to run code anyway

Well that's true, then probably my point is negligible.